src/libs/actions/Policy.js

```diff
@@ -751,6 +752,55 @@ function clearAddMemberError(policyID, memberEmail) {
     });
 }
 
+/**
+ * @param {String} value
+ * @returns {String}
+ */
+function capitalizeFirstLetter(value) {
```

Comment:
It does not really feel like this function belongs here.. more importantly, this function already exists: https://github.com/Expensify/expensify-common/blob/main/lib/str.js#L327 -- please change the code to use that instead 🙇

Reply:
Was this updated?

Reply:
[This PR](https://github.com/Expensify/App/pull/10728) was created to remove the out of place method